src/azure-cli/azure/cli/command_modules/serviceconnector/_help.py

```diff
@@ -47,7 +47,7 @@ def get_auth_info_params(auth_type):
         AUTH_TYPE.Secret: '--secret name=XX secret=XX',
         AUTH_TYPE.SecretAuto: '--secret',
         AUTH_TYPE.SystemIdentity: '--system-identity',
-        AUTH_TYPE.ServicePrincipalSecret: '--service-principal client-id=XX secret=XX',
+        AUTH_TYPE.ServicePrincipalSecret: '--service-principal client-id=XX object-id=XX secret=XX',
```

Comment:
object_id is optional right?

Reply:
yes, it's optional.
- add this to work around a CLI linter bug https://github.com/Azure/azure-cli-dev-tools/issues/331
- in argument help, it's still optional
![image](https://user-images.githubusercontent.com/62928370/165698744-1bfaa6f5-7d3f-4f4d-a6a1-064a7a6d1c1e.png)
- it only appears in aks+keyvault command example